### DISCUSSION TITLE
Edit the text in the fields for delivery address

### DISCUSSION THREAD

**Discussion by peerrott** [woocommerce/woocommerce#45808] [Checkout Flow]
It would be a big advantage to be able to edit the texts in the different fields within delivery adress. In my case I would like to rename Appartment,suite.. to VAT number.
Perhaps is it the pipeline?

**Comment by senadir** [comment]
I'm not sure I understand, do you want to make the apartment field behave a like it's VAT field? this should be doable with code.

**Comment by peerrott** [comment]
Hi Seghir
Thank you for your reply.
I just wanted to replace the predefined texts in one of the fields, but I have knowledge in coding.
Are there any plans to update Woo to make this possible?
Or are you able to guide me in how to with coding?

Best regards
Peer Rott

24 mars 2024 kl. 00:33 skrev Seghir Nadir ***@***.***>:

﻿

I'm not sure I understand, do you want to make the apartment field behave a like it's VAT field? this should be doable with code.

**Comment by senadir** [comment]
```php
add_filter(
	'woocommerce_get_country_locale_base',
	function ( $fields ) {
		$fields['address_2']['label'] = __( 'VAT number', 'domain' );
		return $fields;
	},
	10,
	1
);
```
